pages/index.tsx

```diff
@@ -130,15 +132,23 @@ export const HomePage: NextPage = () => {
             value={keyword}
             placeholder="Search by keyword"
             onChange={(event) => setKeyword(event.target.value)}
-            onKeyDown={(event) => onEnter({ event, handler: filterResources })}
           />
         </InputGroup>
-        <Button onClick={filterResources}>Search</Button>
       </Stack>
       <Stack spacing="32px" px="112px" py="64px">
-        <Stack spacing="16px">
-          <Button onClick={onDrawerOpen}>View your list</Button>
-        </Stack>
+        <Button onClick={onDrawerOpen}>View your list</Button>
+        <Select
+          isMulti
+          isSearchable
+          options={allNeeds?.map((need) => ({
```

Comment:
would it be beneficial to have a fallback option object for when ``alNeeds`` is not defined? Or does ``Select`` handle undefined values?

Reply:
Hmm, if `allNeeds` isn't defined, the select just wouldn't have any options to filter by but I don't think it would cause any errors.

We could hardcode the list of needs as a fallback because I don't think they change that much and right now there's only 24 of them but I think getting them via an API call keeps us more resilient to changes.